.github/workflows/run-integration-tests.yml

```diff
@@ -82,11 +75,8 @@ jobs:
             COMPOSER_NO_INTERACTION: 1
         strategy:
             matrix:
-                php: [7.4, 7.3, 7.2]
-                laravel: [8.*, 7.*, 6.*]
-                exclude:
-                  - laravel: 8.*
-                    php: 7.2
+                php: [8.0, 7.4, 7.3]
```

Comment:
Adding `8.0` here is actually a bug fix, it was forgotten in https://github.com/barryvdh/laravel-ide-helper/pull/1106